src/views/credentials/viewCredentialsList.tsx

```diff
@@ -94,7 +94,11 @@ const CredentialsListView: React.FunctionComponent = () => {
           context: 'deleted-credential',
           name: pendingDeleteCredential?.name
         });
-        addAlert(successMessage, 'success', getUniqueId());
+        addAlert({
+          title: successMessage,
+          variant: 'success',
+          key: getUniqueId()
```

Comment:
required: we switch from using key to id just flip `key` to `id` in all of the related view files
- viewCredentials
- viewScansList
- viewSourcesList